client/gutenberg/extensions/presets/jetpack/editor-shared/jetpack-plugin-sidebar.scss

```diff
@@ -0,0 +1,40 @@
+.edit-post-pinned-plugins,
+.edit-post-more-menu__content {
+	.components-icon-button {
+		.jetpack-logo {
+			width: 20px;
+			height: 20px;
+		}
+	}
+}
+
+.edit-post-more-menu__content {
+	.components-icon-button {
+		.jetpack-logo {
+			margin-right: 4px;
+		}
+	}
+}
+
+.edit-post-pinned-plugins {
+	.components-icon-button {
+		&:not( .is-toggled ),
+		&:hover,
+		&.is-toggled,
+		&.is-toggled:hover {
+			.jetpack-logo,
+			.jetpack-logo * {
+				fill: none !important;
+				stroke: none !important;
+			}
+
+			.jetpack-logo__icon-circle {
+				fill: #00be28 !important;
+			}
+
+			.jetpack-logo__icon-triangle {
+				fill: #fff !important;
```

Comment:
Should we use a named color here?

Reply:
https://github.com/Automattic/wp-calypso/blob/d0a2b47e1935a2388783621e1ccd5a8e9fe62102/assets/stylesheets/shared/_colors.scss#L52-L53
https://github.com/Automattic/color-studio/blob/302bab1a6279ce539cd47cf369bf307a0d300d71/dist/colors.scss#L3

Reply:
👍 e9dfd4e10f1f8c0107652830fcd201b2aa6ac6a5